changelog/30320.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+database: Fix a bug where static roles created in a prior version get rotated on backend restart.
```

Comment:
nit: since this is a bug there's probably no need to mention it again. Also you may want to mention which versions are affected. E.g: version <= x.x.x  - or something like that.